server/db/models.py

```diff
@@ -158,25 +179,72 @@
+   Column('reporter_id',            ForeignKey('login.id'),         nullable=False),
+   Column('report_description',     Text),
+   Column('game_id',                ForeignKey('game_stats.id'),    index=True),
    Column('game_incident_timecode', String(100)),
-   Column('moderator_notice', Text),
+   Column('moderator_notice',       Text),
    Column('moderator_private_note', Text),
-   Column('last_moderator', ForeignKey('login.id'), index=True),
-   Column('create_time',   TIMESTAMP,  nullable=False),
-   Column('update_time',   TIMESTAMP,  nullable=False)
+   Column('last_moderator',         ForeignKey('login.id'),         index=True),
+   Column('create_time',            TIMESTAMP,                      nullable=False),
+   Column('update_time',            TIMESTAMP,                      nullable=False),
+   Column(
+       'report_status',
+       Enum('AWAITING', 'PROCESSING', 'COMPLETED', 'DISCARDED'),
+       nullable=False
+    )
 )
 
 
 reported_user = Table(
    'reported_user', metadata,
-   Column('id', Integer, primary_key=True),
-   Column('player_id', ForeignKey('login.id'), nullable=False),
-   Column('report_id', ForeignKey('moderation_report.id'), nullable=False),
-   Column('create_time',   TIMESTAMP,  nullable=False),
-   Column('update_time',   TIMESTAMP,  nullable=False)
+   Column('id',             Integer,                            primary_key=True),
+   Column('player_id',      ForeignKey('login.id'),             nullable=False),
+   Column('report_id',      ForeignKey('moderation_report.id'), nullable=False),
+   Column('create_time',    TIMESTAMP,                          nullable=False),
+   Column('update_time',    TIMESTAMP,                          nullable=False)
+)
+
+teamkills = Table(
+    'teamkills', metadata,
+    Column('id',            Integer, primary_key=True),
+    Column('teamkiller',    Integer, ForeignKey('login.id')),
+    Column('victim',        Integer, ForeignKey('login.id')),
+    Column('game_id',       Integer, ForeignKey('game_stats.id')),
+    Column('gametime',      Integer),
+)
+
+table_mod = Table(
+    'table_mod', metadata,
+    Column('id',            Integer,    nullable=False),
+    Column('uid',            String,     nullable=False),
+    Column('name',          String,     nullable=False),
+    Column('version',       Integer,    nullable=False),
+    Column('author',        String(100), nullable=False),
+    Column('ui',            Integer),
+    Column('date',          TIMESTAMP,  nullable=False),
+    Column('downloads',     Integer),
+    Column('likes',         Float),
+    Column('played',        Integer),
+    Column('description',   String,     nullable=False),
+    Column('filename',      String,     nullable=False),
+    Column('icon',          String),
+    Column('likers',        LONGBLOB),
+)
+
+mod_stats = Table(
+    'mod_stats', metadata,
+    Column('mod_id',        Integer,    ForeignKey('mod.id'), primary_key=True),
```

Comment:
Is it supposed to be `table_mod`?
```suggestion
    Column('mod_id',        Integer,    ForeignKey('table_mod.id'), primary_key=True),
```

Reply:
I was looking at this:
http://faforever.github.io/db/tables/mod_stats.html
where the parent is `mod`, not `table_mod` - but I was rather confused why there is both, too.

Reply:
Oh, `table_mod` is a view and `mod` is the actual table. Great choice of names...